doc/user/content/sql/show-types.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "SHOW TYPES"
+description: "`SHOW TYPES` returns a list of the data types in your Materialize instance."
+menu:
+  main:
+    parent: 'sql'
+---
+
+`SHOW TYPES` returns a list of the data types in your Materialize instance. By default, only user-created types are returned.
+
+## Syntax
+
+{{< diagram "show-types.svg" >}}
+
+Field | Use
+------|-----
+`EXTENDED` |  Returns system types as well as user-created types. By default, only user-created types are returned.
+`FULL`| Returns the creator of the data type (user or system).
+
+## Examples
+
+### Show custom data types
```

Comment:
I think examples make the most sense if you show users how to get from tabula rasa to the state you're showing them; in this case, I'd include the statement to create the types. Don't necessarily need as many.